firmware/boron-ins-fsm/test/mocks/mock_serial.h

```diff
@@ -4,37 +4,26 @@
 
 #pragma once
 
-#define Serial 0                                // not implemented
-#define Serial1 __fetch_mock_global_Serial1()   // defined as mock USART Serial
+#define Serial  0                              // not implemented
+#define Serial1 __fetch_mock_global_Serial1()  // defined as mock USART Serial
 
-#define SERIAL_8N1 0                            // fake definition
+#define SERIAL_8N1 0  // fake definition
 
 // Fake class for USARTSerial
-class MockUSARTSerial
-{
-public:
+class MockUSARTSerial {
+   public:
     MockUSARTSerial() {}
 
-public:
-    void begin(unsigned long baud, uint32_t config) {
-        return;
-    }
+   public:
+    void begin(unsigned long baud, uint32_t config) { return; }
 
-    int read(void) {
-        return -1;
-    }
+    int read(void) { return -1; }
```

Comment:
Oh, I don't really like this style. I prefer where the body of the function is on its own line. Is this a clang config that we can change?

Reply:
Hmm, looks like version 12 had some different config settings. This should be fixed now that I changed the value of AllowShortFunctionsOnASingleLine.